script/check-new-links.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+NOT_OK_COUNT=0
+SKIP_PATTERNS=(
+	'github\.com' # links may not exist yet
```

Comment:
Small and not breaking: This seems dangerously broad, we want to check only for the editing links on branches. The utilised RegEx below is `/github.com/(.*)/edit/`. Can't we pour that into `grep` instead?

Reply:
Sure.

It would be `github.com/.*/edit` as we don't have the bounding slashes in this context, nor are we capturing the wildcard matching in the middle.

(I'm not sure why the capturing is there in the (unchanged) original check which we run as a cron-job ... or if it doesn't mean capturing in that context.)

In "About" the github check was relaxed and I don't know the case which caused that. As such, if we tighten this back up, we may not be able to copy-paste this over to About without modification, but that's totally fine.